analyzer/testdata/src/gocritic/f1.go

```diff
@@ -442,3 +442,22 @@ func rangeExprCopy() {
 		}
 	}
 }
+
+func badCond(x, y int) {
+	if x < -10 && x > 10 { // want `\Qthe condition is always false because -10 <= 10`
```

Comment:
can we catch a case, when we've different consts? ex: -8 and 9 respectively.

Reply:
Sure, we have a test that also does some arithmetics:
```go
	const ten = 10
	if x > ten+1 && x < -10 { // want `\Qthe condition is always false because ten+1 >= -10`
	}
```
The `Value().Int()` computes the result of a constexpr.